containers/python/rpc.py

```diff
@@ -306,6 +312,15 @@ def run(self):
 
                         response.send(socket, self.event_history)
 
+                        pred_metric['model_pred_count'] += 1
```

Comment:
Got it. I was thinking about this recently. I deleted `*_time` metrics. 

If we can finalize (at least for now) all the metrics needed to be collected, I will initialize them in the prometheus collector process just once [1] and run update through each dictionary in the pipe. So that every metric record is updated in the metric process (especially matters for histogram and summary data type. 

[1]: the current process initialize a new `gauge` every time the metric is updated.

Reply:
Yeah that's definitely the right way to do it. Let's merge this PR, then you can start on one that tracks the timing metrics with histograms.